src/Graph5/Graph5.tsx

```diff
@@ -0,0 +1,46 @@
+import { AreaChart, Area, ResponsiveContainer } from "recharts";
+import data from "./DataGraph5.json";
+
+function findHighestUV(data: { name: string; uv: number }[]): number {
+	const uvValues = data.map((item) => item.uv);
+	const highestUV = Math.max(...uvValues);
+	return highestUV;
+}
+
+const highestUVValue = findHighestUV(data);
```

Comment:
To bym zrobił chyba w jednej linii tak jak ja to zrobilem w Grafie4, bo tu nazwa zmiennej mowi tak naprawde co to jest, nie robiłbym chyba osobnej funkcji na to

Reply:
done